packages/web/src/instrumentations/console/index.ts

```diff
@@ -5,7 +5,7 @@ export interface ConsoleInstrumentationOptions {
 }
 
 export class ConsoleInstrumentation extends BaseInstrumentation {
-  static defaultDisabledLevels: LogLevel[] = [LogLevel.DEBUG, LogLevel.LOG];
+  static defaultDisabledLevels: LogLevel[] = [LogLevel.DEBUG, LogLevel.TRACE];
```

Comment:
I strongly suggest disabling the `LOG` as well :).

Reply:
The problem with `log` is that there is too much logging happening at this level. Someone may enable it ending up spamming the receiver.